src/lib/logger.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-module.exports = require('./httpd').log;
+const logger = require('../lib/httpd').log;
+const bunyan = require('bunyan');
+
+// disable default bunyan stream to stdout
+logger.level(bunyan.FATAL + 1);
+
+function modifiedStream() {
+  return {
+    write: entry => {
+      const logObject = JSON.parse(entry)
+      // new logging elements
+      logObject['@timestamp'] = logObject.time;
+      logObject['log.level'] = bunyan.nameFromLevel[logObject.level];
+
+      // removing logging elements not required
+      delete logObject.level;
+      delete logObject.v;
+
+      process.stdout.write(JSON.stringify(logObject) + '\n');
```

Comment:
Do we need to do this? I'd suggest if we want to write to stdout, it should be a separate stream dedicated to that.

Reply:
If we moved this out to another stream, then we will need to re-apply the formatting to the logger also?